app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php

```diff
@@ -73,6 +78,15 @@ public function form(Form $form): Form
     {
         return $form
             ->schema([
+                Section::make('Application Name')
+                    ->aside()
+                    ->schema([
+                        TextInput::make('application_name')
+                            ->label('Application Name')
+                            ->required()
+                            ->dehydrated(fn (?string $state): bool => filled($state))
```

Comment:
Is this `dehydrated` still needed? Because this field is now required it would never not be `filled` right?